app/channels/conversation_channel.rb

```diff
@@ -0,0 +1,24 @@
+class ConversationChannel < ApplicationCable::Channel
+  def subscribed
+    reject unless conversation
+    stream_for conversation
+  end
+
+  def speak(data)
+    validate_message(data)
+    Message.create(conversation_id: data['conversation_id'],
```

Comment:
An instance of `Message` will be attempt to be created even if `validate_message(data)` goes wrong.

Maybe you can break the flow if the message data is not correct